recipes/bc/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set pkgname = "bc" %}
+{% set version = "1.06" %}
+{% set sha1 = "c8f258a7355b40a485007c40865480349c157292" %}
+{% set md5 = "d44b5dddebd8a7a7309aea6c36fda117" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: http://ftpmirror.gnu.org/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+  md5: {{ md5 }}
+  patches:
+    - verify_checklib_results.sh.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
```

Comment:
You may need to to add a few build dependencies that are not available on the docker image by default, like `flex`.